src/invidious/frontend/comments_reddit.cr

```diff
@@ -33,7 +33,7 @@ module Invidious::Frontend::Comments
             <a href="javascript:void(0)" data-onclick="toggle_parent">[ − ]</a>
             <b><a href="https://www.reddit.com/user/#{child.author}">#{child.author}</a></b>
             #{translate_count(locale, "comments_points_count", child.score, NumberFormatting::Separator)}
-            <span title="#{child.created_utc.to_s(translate(locale, "%a %B %-d %T %Y UTC"))}">#{translate(locale, "`x` ago", recode_date(child.created_utc, locale))}</span>
+            <span title="#{child.created_utc.to_s("%a %B %-d %T %Y UTC")}">#{translate(locale, "`x` ago", recode_date(child.created_utc, locale))}</span>
```

Comment:
The title here should still be translated.  A computer-readable timestamp should be placed under the `<time>` element

Reply:
you mean in the default time format for the locale? sadly I don't see a way to print that in https://crystal-lang.org/api/1.11.2/Time/Format.html maybe I'm missing something. this time is in UTC rather than user's timezone anyway.

Reply:
Yeah, the Crystal stdlib is quite limited in that regard. For now, we should leave date & numbers formatting alone, until a proper I18n/CLDR shard exists in Crystal.

Reply:
I need to start working on Lens again